hieradata/development.yaml

```diff
@@ -163,6 +163,8 @@ govuk::apps::signon::enable_procfile_worker: false
 govuk::apps::smartanswers::expose_govspeak: true
 govuk::apps::specialist_publisher::enable_procfile_worker: false
 govuk::apps::specialist_publisher::publish_pre_production_finders: true
+govuk::apps::specialist_publisher::mongodb_nodes: ['localhost']
+govuk::apps::specialist_publisher::mongodb_name: 'specialist_publisher_development'
```

Comment:
I'm a little concerned that the one below is apparently a production database, and is different to this one. Surely they should be using the same db_name?